x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -344,13 +344,23 @@ export const AddExceptionFlyoutWrapper: React.FC<AddExceptionFlyoutWrapperProps>
     return ruleIndices;
   }, [enrichedAlert, ruleIndices]);
 
+  const useRuleDataViewId = useMemo(() => {
```

Comment:
nit: maybe just name this dataViewId, prefacing it with use makes it look like a custom hook at first glance

Reply:
yup good catch